### PR TITLE
[docs][dagster-fivetran] Add Fivetran getting started guide

### PR DESCRIPTION
**Pull request by benpankow**
## Summary

Introduces a Fivetran getting started guide, very similar to the existing Airbyte one. Steps through defining a Fivetran resource and then automatically loading assets or constructing manually.

### CODE REVIEW

**Comment by benpankow** [comment]
Current dependencies on/for this PR:
* master
  * **PR #10468** <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/10468" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/dagster-io/dagster/10468?utm_source=stack-comment).

File: docs/content/integrations/fivetran.mdx [hunk 1]
```diff
@@ -0,0 +1,209 @@
+---
+title: "Using Dagster with Fivetran"
+description: Integrate your Fivetran connectors into Dagster.
+---
+
+# Using Fivetran with Dagster
+
+Dagster can orchestrate your Fivetran connectors, making it easy to chain an Fivetran sync with upstream or downstream steps in your workflow.
```
Comment (erinkcochran87) [inline docs/content/integrations/fivetran.mdx:8]:
```suggestion
Dagster can orchestrate your Fivetran connectors, making it easy to chain a Fivetran sync with upstream or downstream steps in your workflow.
```

File: docs/content/integrations/fivetran.mdx [hunk 2]
```diff
@@ -0,0 +1,209 @@
...
+    width={2301}
+    height={1232}
+  />
+</center>
+
+---
+
+## Fivetran connectors and Dagster software-defined assets
+
+An [Fivetran connector](https://fivetran.com/docs/getting-started/core-concepts) defines a series of schemas and tables which are synced between a source and a destination. During a sync, data is retrieved from the source and written to the destination as one or more tables. Dagster represents each of the replicas generated in the destination as a software-defined asset. This enables you to easily:
```
Comment (erinkcochran87) [inline docs/content/integrations/fivetran.mdx:27]:
```suggestion
A [Fivetran connector](https://fivetran.com/docs/getting-started/core-concepts) defines a series of schemas and tables which are synced between a source and a destination. During a sync, data is retrieved from the source and written to the destination as one or more tables. Dagster represents each of the replicas generated in the destination as a software-defined asset. This enables you to easily:
```

File: docs/content/integrations/fivetran.mdx [hunk 3]
```diff
@@ -0,0 +1,209 @@
...
+
+- Visualize the schemas and tables involved in an Fivetran connector and execute a sync from Dagster
```
Comment (erinkcochran87) [inline docs/content/integrations/fivetran.mdx:29]:
```suggestion
- Visualize the schemas and tables involved in a Fivetran connector and execute a sync from Dagster
```

File: docs/content/integrations/fivetran.mdx [hunk 4]
```diff
@@ -0,0 +1,209 @@
...
+
+You'll also want to have set up [a Fivetran instance](https://fivetran.com/docs/getting-started) and have a Fivetran [API key and API secret](https://fivetran.com/docs/rest-api/getting-started), which can be generated by a Fivetran Account Administrator.
+
+---
+
+## Step 1: Connecting to Fivetran
+
+The first step in using Fivetran with Dagster is to tell Dagster how to connect to your Fivetran instance using an Fivetran [resource](/concepts/resources). This resource contains the credentials needed to access your Fivetran instance.
+
+We will supply our credentials as environment variables:
```
Comment (erinkcochran87) [inline docs/content/integrations/fivetran.mdx:52]: Should we add a link to the new env vars + secrets guide in here somewhere? https://docs.dagster.io/guides/dagster/using-environment-variables-and-secrets

File: docs/content/integrations/fivetran.mdx [hunk 5]
```diff
@@ -0,0 +1,209 @@
...
+The `load_assets_from_fivetran_instance` function retrieves all of the connectors you have defined in the Fivetran interface, creating software-defined assets for each generated table. Each connector has an associated [op](https://docs.dagster.io/concepts/ops-jobs-graphs/ops#ops) which triggers a sync of that connector.
+
+</TabItem>
+<TabItem name="Manually building assets">
+
+### Manually building Fivetran asset definitions
+
+Instead of having Dagster automatically create the asset defintions for your Fivetran instance, you can opt to individually build them. First, determine the Connector IDs for each of the connectors you would like to build assets for. The Connector ID can be found on the Setup page for each connector in the Fivetran UI:
+
+<center>
```
Comment (erinkcochran87) [inline docs/content/integrations/fivetran.mdx:107]: This image is super stretchy when viewed in preview ([link](https://dagster-git-benpankow-fivetran-docs-elementl.vercel.app/integrations/fivetran#manually-building-fivetran-asset-definitions)) - I'm not sure if it's the source image that's like this or if the pixel sizes here are incorrect. Might need to re-run `make mdx-format`

File: docs/content/integrations/fivetran.mdx [hunk 6]
```diff
@@ -0,0 +1,209 @@
...
+
+fivetran_assets = build_fivetran_assets(
+    connector_id="omit_constitutional",
+    destination_tables=["public.survey_responses", "public.surveys"],
+)
+```
+
+#### Adding a resource
+
+Manually built Fivetran assets require an `fivetran_resource`, which defines how to connect and interact with your Fivetran instance.
```
Comment (erinkcochran87) [inline docs/content/integrations/fivetran.mdx:129]:
```suggestion
Manually-built Fivetran assets require a `fivetran_resource`, which defines how to connect and interact with your Fivetran instance.
```

**Review by erinkcochran87 — APPROVED** [review]
This is looking really good! I noticed some small typos and added suggestions to fix those, along with an issue with a screenshot.

One other quick ask - could you add a link to this in the **Integration guides** section here?: https://dagster-git-benpankow-fivetran-docs-elementl.vercel.app/integrations#integration-guides

File: docs/content/integrations/fivetran.mdx (hunk 4 above)
Comment (benpankow) [reply docs/content/integrations/fivetran.mdx:52]: Good idea, added 👍

File: docs/content/integrations/fivetran.mdx (hunk 5 above)
Comment (benpankow) [reply docs/content/integrations/fivetran.mdx:107]: Fixed!

**Pull request merged**